rfcs/0167-Easy-Taskcluster-setup.md

```diff
@@ -0,0 +1,44 @@
+# RFC 169 - Easy Taskcluster Setup
+* Comments: [#169](https://github.com/taskcluster/taskcluster-rfcs/pull/169)
+* Proposed by: @sarah-clements
+
+# Summary
+
+Make Taskcluster easier to setup into a repository
+
+## Motivation
+
+Currently setting up a Taskcluster config file is confusing compared to its competitors. The [Enhance Taskgraph RFC](https://github.com/mozilla-releng/releng-rfcs/pull/36) will simplify the configuration file needed to get started, but we can go a step further and create a user-friendly experience that will guide the user through setup.
+
+## Assumptions
+
+The solution can be repository host agnostic, but the details of this RFC are based on the Taskcluster integration with Github.
+
+
+# Details
+
+Create a tool or feature that will automatically generate a yaml config file as part of the taskcluster-github app installation and setup. This tool would have presets for common use cases such as running javascript tests or python tests. This might look like a series of questions in a form to help users get started with some default scope, worker type configurations and basic commands.
```

Comment:
I _think_ things like scopes and worker types should probably be abstracted away from the user here. I suspect that they won't need to know anything about scopes (we can probably guess them accurately for the cases we're addressing here), and "worker type" can be assumed by looking at the target platforms the user wants to build for.

Reply:
Yeah, it seems like the guided setup would cover a lot of scenarios but I'm wondering if there are any "advanced use cases" where we at least want to present some documentation in a clearer way (at least on worker type) for custom configuration. Scopes seems to be well-documented but there is a lot granularity that can be overwhelming.